.github/workflows/client.yml

```diff
@@ -25,6 +25,10 @@ jobs:
         k8s:
           - v1.21
           - v1.26
+        tls:
+          - "openssl-tls"
+          - "rustls-tls"
+    name: "local (k8s ${{ matrix.k8s }}, ${{ matrix.tls }})"
     timeout-minutes: 15
```

Comment:
It looks like this timeout is restrictive in this setup... Unfortunately we're going to have to increase this...